be2-scala/src/test/scala/ch/epfl/pop/storage/FakeSecurityModuleActor.scala

```diff
@@ -0,0 +1,36 @@
+package ch.epfl.pop.storage
+
+import akka.actor.Actor
+import akka.event.LoggingReceive
+import ch.epfl.pop.storage.FakeSecurityModuleActor.{rsaPrivateKey, rsaPublicKey, rsaPublicKeyPem}
+import ch.epfl.pop.storage.SecurityModuleActor.{ReadRsaPublicKey, ReadRsaPublicKeyAck, ReadRsaPublicKeyPem, ReadRsaPublicKeyPemAck, SignJwt, SignJwtAck}
+import com.auth0.jwt.algorithms.Algorithm
+
+import java.security.interfaces.{RSAPrivateKey, RSAPublicKey}
+import java.security.{KeyPair, KeyPairGenerator}
+
+final case class FakeSecurityModuleActor() extends Actor {
```

Comment:
I am not sure of the relevance of this entire fake actor. You repeat some of the functionalities. Is there not a simpler way to do the similar things?

Reply:
Indeed I removed it from the project as it would also be hard to maintain because it could not inherit `SecurityModuleActor` which is sealed for security.